app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/SearchResultsAdapter.java

```diff
@@ -44,54 +44,21 @@ public SearchResultHolder onCreateViewHolder(ViewGroup parent, int viewType) {
 
     @Override
     public void onBindViewHolder(final SearchResultHolder holder, int position) {
-        Datum datum = datumList.get(position);
+        final Datum datum = datumList.get(position);
+        holder.previewImageView.setVisibility(View.GONE);
         if (datum != null) {
             holder.titleTextView.setText(Html.fromHtml(datum.getTitle()));
             holder.descriptionTextView.setText(Html.fromHtml(datum.getDescription()));
-            if (!TextUtils.isEmpty(datum.getLink())) {
-                LinkPreviewCallback linkPreviewCallback = new LinkPreviewCallback() {
```

Comment:
I didn't exactly get what you did here. Like, why is there a need to remove this code. This code is used to show preview of the links. Why are we removing it?